module 4/assignment4.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Assignment 4</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="_css/mowen_a4.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+    <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+    <script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+    ]<link rel="stylesheet" type="text/css" href="_css/bistro.css">
+</head>
+<body>
+<!-- Randall Mowen
+     assignment4.html
+     INFO2439
+     Thoendel
+     10/5/19
+-->
+
+<div data-role="page" id="assignment4"> <!-- opening page div -->
+    <header>
+    <div data-role="header" data-position="fixed" data-id="header" class="top-header" data-theme="a"><!-- opening header div -->
+        <a href="../index.html" data-icon="home" rel="external">Home</a>
+        <h1>Randall Mowen</h1>
+        </div><!-- closing header div -->
+    </header>
+
+    <div data-role="main" class="ui-content"> <!-- opening content div -->
+        <form action="http://emthoendel.faculty.mccinfo.net/2439/bistrosurvey_insert.php" method="post" id="form1">
+            <p>How would you rate your meal today?</p>
+            <select name="meal_rate" id="meal_rate">
+                <option value="Poor">1-Poor</option>
+                <option value="Needs_Work">2-Needs Work</option>
+                <option value="Fair">3-Fair</option>
+                <option value="Good">4-Good</option>
+                <option value="Excellent">5-Excellent</option>
+            </select>
```

Comment:
Your book shows the use of <div> tags to group each label + input pair to focus enhancements and allow for easier maintenance.